certified-connectors/ShareEffect/apiDefinition.swagger.json

```diff
@@ -145,8 +142,8 @@
                   "type": "string",
                   "x-ms-dynamic-values": {
                     "operationId": "GetTermGroups",
-                    "value-path": "key-item-output",
-                    "value-title": "key-item-output"
+                    "value-path": "",
+                    "value-title": ""
```

Comment:
@Rens-Braspenning Please remove the empty value properties i.e. "value-path" and "value-title"